third_party/polymer.bzl

```diff
@@ -1379,25 +1425,17 @@ def tensorboard_polymer_workspace():
   web_library_external(
       name = "org_polymer_webcomponentsjs",
       licenses = ["notice"],  # BSD-3-Clause
-      sha256 = "2164e6d07d72d4326865ab7c0acee5493870e1ae5f31e9d22665dff4a4773078",
+      sha256 = "c3a4082a1f6930fb149191930c05a46ddaa5c272a7d0bfa9c3af413b858d1eb2",
       urls = [
-          "https://mirror.bazel.build/github.com/webcomponents/webcomponentsjs/archive/v0.7.24.tar.gz",
-          "https://github.com/webcomponents/webcomponentsjs/archive/v0.7.24.tar.gz",
+          # 2.x only distribute via source code (as opposed to built binary) and
```

Comment:
What does this mean? The `webcomponents-lite.js` file does appear to be
minified.

Reply:
Skimming through [1], it looks very incompatible with Vulcanization and lines like [2] was a big concern.

[1]: https://github.com/webcomponents/webcomponentsjs/blob/v2.2.10/webcomponents-loader.js
[2]: https://github.com/webcomponents/webcomponentsjs/blob/v2.2.10/entrypoints/webcomponents-sd-index.js#L17-L18

I am a bit lost by your second sentence: does webcomponents 2.x release contain that file? I can't seem to find it in their source. Is it part of their npm package?

Reply:
I was talking about `webcomponentjs-1.3.3/webcomponents-lite.js`, which
appears in the tarball fetched by this web library. If this comment is
referring to a future release rather than this distribution, why is it
here? (I presumed that the “2.x” referred to Polymer 2.)

Reply:
Ah, I will clear that up. I was trying to update most of our Polymer dependencies except I couldn't do it for this one and wanted to say why.